downstream/modules/platform/proc-edge-manager-update-os-cli.adoc

```diff
@@ -0,0 +1,41 @@
+[id="edge-manager-update-os-cli"]
+
+= Updating the operating system on the CLI
+
+Update a device using the CLI.
+
+Complete the following steps:
+
+.Procedure
+
+. Get the current resource manifest of the device by running the following command:
+
++
+[source,bash]
+----
+flightctl get device/<device_name> -o yaml > my_device.yaml
+----
+
+. Edit the `Device` resource to specify the new operating system name and version target.
```

Comment:
```suggestion
. Edit the `spec.os` block of the`Device` resource to specify the new operating system name and version target.
```
It might be handy to specify exactly which block in the YAML file that's updated.